website/versioned_docs/version-3.x/tab-navigator.md

```diff
@@ -0,0 +1,126 @@
+---
+id: version-3.x-tab-navigator
+title: createTabNavigator
+sidebar_label: createTabNavigator
```

Comment:
createTabNavigator is gone in react-navigation@>=3.0, we have `createBottomTabNavigator` and `createMaterialTopNavigator` instead now :) so we can remove this doc

Reply:
Got it, makes sense that the file wasn't there already then!